### PR TITLE
Feat/449 mgmt project oneliner

### LINKED ISSUE #449: Mgmt: Adjust Project OneLiner
Add a new text-field with which users can change the one-liner of their project.

### PR DESCRIPTION
**Pull request by marhali**
Resolves #449

### CODE REVIEW

File: components/app/account/project/ProjectDescription.vue [hunk 1]
```diff
@@ -1,13 +1,14 @@
 <template>
   <div class="shadow rounded p-4">
-    <p class="text-primary small-text">
-      {{ $t('manage.project.description') }}
-    </p>
+    <label class="label">
+      <span class="label-text text-primary">{{
```
Comment (KinimodEtlon) [inline components/app/account/project/ProjectDescription.vue:4]: nitpick(blocking): the classes can be moved inside the label div

File: components/app/account/project/socials/ProjectSocials.vue [hunk 2]
```diff
@@ -1,9 +1,11 @@
 <template>
   <div class="shadow rounded p-4">
     <div class="flex justify-between">
-      <p class="small-text text-primary">
-        {{ $t('manage.project.socials.title') }}
-      </p>
+      <label class="label">
+        <span class="label-text text-primary">
```
Comment (KinimodEtlon) [inline components/app/account/project/socials/ProjectSocials.vue:5]: nitpick(blocking): same as above

File: mixins/Constants.js [hunk 3]
```diff
@@ -6,6 +6,7 @@ export default {
       PICTURE_MAX_SIZE: 4000000, // 4MB
       PICTURE_MAX_COUNT: 8,
       PICTURE_ALLOWED_TYPES: 'image/jpeg,image/svg+xml,image/png',
-    }
+      PROJECT_ONE_LINER_MAX_LENGTH: 80,
```
Comment (KinimodEtlon) [inline mixins/Constants.js:9]: question(non-blocking): we should align this with the database, right now we allow a max length of 2048, which is probably to much. How many chars can we display on the cards?

File: pages/management/index.vue [hunk 4]
```diff
@@ -130,7 +132,8 @@ export default Vue.extend({
         {
           hid: 'description',
           name: 'description',
-          content: 'Login with your favorite wallet provider to manage your project or proposal.',
+          content:
+            'Login with your favorite wallet provider to manage your project or proposal.',
```
Comment (KinimodEtlon) [inline pages/management/index.vue:136]: nitpick(blocking): use a const or even better a i18n variable. I also requested this in another pr, just skip it here if you already did this.

**Review by KinimodEtlon — CHANGES_REQUESTED** [review]

**Review by Abrom8 — APPROVED** [review]

File: pages/management/index.vue (hunk 4 above)
Comment (marhali) [reply pages/management/index.vue:136]: Created an ticket for this to align this up for the entire project. See #458

File: mixins/Constants.js (hunk 3 above)
Comment (marhali) [reply mixins/Constants.js:9]: As we discussed internally we will stick on 80 characters for now. Import script and database will continue to support larger values.

**Review by KinimodEtlon — APPROVED** [review]

**Pull request merged**